libraries/src/Helper/TagsHelper.php

```diff
@@ -825,7 +825,7 @@ public static function getTypes($arrayType = 'objectList', $selectTypes = null,
      *
      * @deprecated  5.3 will be removed in 7.0
      */
-    public function postStoreProcess(TableInterface $table, $newTags = [], $replace = true)
+    public function postStoreProcess(TableInterface $table, array $newTags = [], bool $replace = true): void
     {
         $this->postStore($table, $newTags, $replace);
```

Comment:
Since this method already exists, we are not allowed to change the signature.

```suggestion
    public function postStoreProcess(TableInterface $table, $newTags = [], $replace = true): void
    {
        @trigger_error('7.0 Method postStoreProcess() is deprecated, use postStore() instead.', \E_USER_DEPRECATED);

        $this->postStore($table, (array) $newTags, (bool) $replace);
    }
```

Also please update the deprecation message with the replacement syntax.